src/Uno.UI/UI/Xaml/Controls/Flyout/Flyout.cs

```diff
@@ -88,23 +86,22 @@ private void OnPopupOpened(object sender, object e)
 		{
 			var child = _popup.Child as FrameworkElement;
 
-			if(child != null)
-			{
-				SizeChangedEventHandler handler = (_, __) => SetPopupPositionPartial(Target);
-
-				child.SizeChanged += handler;
-
-				_sizeChangedDisposable.Disposable = Disposable
-					.Create(() => child.SizeChanged -= handler);
-			}
+			if (child != null)
+				child.SizeChanged += OnChildSizeChanged;
 		}
 
+		private void OnChildSizeChanged(object sender, SizeChangedEventArgs args) => SetPopupPositionPartial(Target);
+
 		partial void InitializePartial();
 
 		private void OnPopupClosed(object sender, object e)
 		{
+			var child = _popup.Child as FrameworkElement;
+
+			if (child != null)
+				child.SizeChanged -= OnChildSizeChanged;
```

Comment:
```suggestion
			{
				child.SizeChanged -= OnChildSizeChanged;
			}
```

Reply:
Find this convention overkill. Mostly half the code blocks are one liners, more so in well refactored code 

Might make it easier for code reviews/compare and merge but it simply add to much useless lines of code

i'll follow the convention but ..